wcivf/apps/elections/models.py

```diff
@@ -467,6 +472,20 @@ def postal_vote_application_deadline(self):
             self.ballot_paper_id, self.post.territory
         ).postal_vote_application_deadline
 
+    @property
+    def postal_vote_requires_form(self):
```

Comment:
This logic can be replaced by the matcher code. See example here: https://github.com/DemocracyClub/uk-election-ids/blob/master/tests/test_meta_data_matcher.py#L30